integration_tests/enum_04.py

```diff
@@ -0,0 +1,71 @@
+from ltypes import i32
+from enum import Enum, auto
+
+class Color(Enum):
+    RED: i32 = auto()
+    GREEN: i32 = auto()
+    BLUE: i32 = 7
+    YELLOW: i32 = auto()
+    WHITE: i32 = auto()
+    PINK: i32 = 15
+    GREY: i32 = auto()
+
+class Integers(Enum):
+    a: i32 = -5
+    b: i32 = auto()
+    c: i32 = auto()
+    d: i32 = 0
+    e: i32 = auto()
+    f: i32 = 7
+    g: i32 = auto()
```

Comment:
Python does not have the best syntax here, but since this is what they do, we'll do it too for now. We can later fix this to something like:
```python
@enum
class Color:
    RED: i32
    GREEN: i32
    BLUE: i32 = 7
    YELLOW: i32
    WHITE: i32
    PINK: i32 = 15
    GREY: i32
```
which seems nicer.